jsk_perception/sample/sample_blob_detector.launch

```diff
@@ -0,0 +1,40 @@
+<launch>
+
+  <arg name="gui" default="true" />
+
+  <node name="image_publisher"
+        pkg="jsk_perception" type="image_publisher.py">
+    <param name="file_name" value="$(find jsk_perception)/sample/image/mask0001.jpg" />
```

Comment:
how did you make this image? computer generated? or copied from somewhere? I prefer to use more descriptive name